near/omni-bridge/src/lib.rs

```diff
@@ -875,6 +1022,14 @@ impl Contract {
     pub fn get_current_destination_nonce(&self, chain_kind: ChainKind) -> Nonce {
         self.destination_nonces.get(&chain_kind).unwrap_or_default()
     }
+
+    #[private]
+    pub fn ft_resolve_transfer(&mut self, amount: U128) -> U128 {
```

Comment:
The naming could be confusing (in the explorer) because the nep141 implementation also has a `ft_resolve_transfer`

Reply:
Renamed 1f38b0469619ad975744e128cc91dc404cde9b6f